velox/exec/HashBuild.cpp

```diff
@@ -700,9 +709,9 @@ bool HashBuild::finishHashBuild() {
     if (spiller_ != nullptr) {
       spillStats += spiller_->stats();
 
-      stats_.spilledBytes = spillStats.spilledBytes;
-      stats_.spilledRows = spillStats.spilledRows;
-      stats_.spilledPartitions = spillStats.spilledPartitions;
+      stats_.spilledBytes += spillStats.spilledBytes;
```

Comment:
Is this a fix for a pre-existing issue?

Reply:
Yes.

Reply:
Got it. It would be nice to extract into a separate PR.